instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py

```diff
@@ -37,7 +37,7 @@
 
 major_version = elasticsearch.VERSION[0]
 
-if major_version == 7:
+if major_version  7:
```

Comment:
Is this a mistake?

Reply:
I think so, I don't know who pushed this change but I will revert it